GVFS/GVFS.Platform.Windows/WindowsPlatform.cs

```diff
@@ -499,6 +499,26 @@ public override HashSet<string> UpgradeBlockingProcesses
 
             // Tests show that 250 is the max supported pipe name length
             public override int MaxPipePathLength => 250;
+
+            public override string UpgradeInstallAdviceMessage
+            {
+                get { return "When ready, run `" + this.UpgradeConfirmCommandMessage + "` from an elevated command prompt."; }

```

Comment:
Why not an interpolated string here? `$"When ready run `{this.UpgradeConfirmCommandMessage}`..."